webui/index.ego

```diff
@@ -19,7 +19,7 @@ func ego_index(w io.Writer, req *http.Request) {
       <span class="dot"></span>
     </span>
   </h3>
-  <div class="interval-slider ltr">
```

Comment:
I believe this was put here because the slider doesn't make sense when displayed as rtl.

Reply:
This is how it looks currently (I just modified english to RTL). If this is incorrect, I'll putt he `ltr` class back:
![](https://i.imgur.com/1QW1EHI.png)

Reply:
Fixed in: https://github.com/contribsys/faktory/pull/360/commits/6c1ab89251f1adf3e1d29b215cef90ae6a1c4883